js/src/modal.js

```diff
@@ -11,7 +11,9 @@ import SelectorEngine from './dom/selector-engine.js'
 import Backdrop from './util/backdrop.js'
 import { enableDismissTrigger } from './util/component-functions.js'
 import FocusTrap from './util/focustrap.js'
-import { defineJQueryPlugin, isRTL, isVisible, reflow } from './util/index.js'
+import {
```

Comment:
I don't see any rule enforcing this formatting upstream in the corresponding commit. I'll try to ask upstream how it could be enforced. In the meantime, I've applied it only on the same files as applied upstream.